packages/wrangler/src/cfetch/internal.ts

```diff
@@ -242,17 +242,16 @@ export async function fetchDashboardScript(
 	if (usesModules) {
 		// Response from edge contains generic "name = worker.js" for dashboard created scripts
 		const form = await response.formData();
-		const entries = Array.from(form.entries());
-		if (entries.length > 1)
-			throw new RangeError("Expected only one entry in multipart response");
-		const [_, file] = entries[0];
+		const files = Array.from(form.entries())
+			.filter(([_, file]) => file instanceof File)
```

Comment:
What is this filter checking? Are there items in the response that aren't parsed as Files?

Reply:
I'm not sure if there will be any non-files or even if that's possible. I was keeping the existing check from L250